### DISCUSSION TITLE
Support icon with validation message

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#8953] [Help]
<img width="211" alt="Screenshot 2023-10-07 at 20 42 04" src="https://github.com/filamentphp/filament/assets/121255432/65210ffc-8081-4a44-ae83-bf4d86d68f6e">

**Comment by leandrocfe** [comment]
Isn't possible I think

**Reply by devhoussam1998** [reply]
currently not possible

**Reply by leandrocfe** [reply]
There is a workaround using hintIcon or suffix
https://github.com/filamentphp/filament/discussions/8373#discussioncomment-6957597

**Reply by devhoussam1998** [reply]
this is not what I'm looking for thanks man

**Reply by krekas** [reply]
Then make a pr for feature that you need.
